GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -176,17 +177,15 @@ void Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat,
 }
 
 // generate vertex array object
-void Mesh::generateVAO(Material::ShaderType key) {
+void Mesh::generateVAO() {
 #if _GVRF_USE_GLES3_
+
+	if (vaoInitiliased_)
+		return;
```

Comment:
Thanks. Didn’t notice this in the diffs.

I will fix and push it soon (along with another Mesh related change).

From: chenchao1407 [mailto:notifications@github.com]
Sent: Thursday, September 03, 2015 2:19 PM
To: Samsung/GearVRf
Cc: Vittalbabu Srinivasan
Subject: Re: [GearVRf] Use single VAO for shaders -- the attribute bind locations for a_position, a_text_coord and a_normal are set to 0, 1 and 2. This allows us to use the same VAO for lit and unlit shaders in Texture Shader. (#236)


In GVRf/Framework/jni/objects/mesh.cpp<https://github.com/Samsung/GearVRf/pull/236#discussion_r38697951>:

>  #if _GVRF_USE_GLES3_

> +

> +     if (vaoInitiliased_)

> +            return;

These two lines are tab based formatting not space based.
Please in Window -> Preference -> C/C++ -> Code Style
And then edit the K&R[Space] profile, change the profile name and for Tab policy, change it from tabs only to spaces only.

Then you do Ctrl + Shift + F, you will see that there is something changed.

Thanks in advance.

—
Reply to this email directly or view it on GitHub<https://github.com/Samsung/GearVRf/pull/236/files#r38697951>.